### ISSUE TITLE
Install ubuntu [dependencies]

### ISSUE THREAD

**Issue by neopostmodern** [Ant1r/ofxPof#3]
I installed pof using deken on Ubuntu 17.04, x86_64

In vanilla pd I somehow can't get it loaded (`pof: can't load library`). I digged around and found the `pof/scripts/ubuntu/install_dependencies.sh` script, considering that maybe it wouldn't load due to unsatisfied dependencies. But the script fails with the following ouput:

```
It seems you are running gcc 5 or later, due to incomatible ABI with previous versions
we need to recompile poco. This will take a while
Press any key to continue... ./install_dependencies.sh: line 176: cd: /home/neopostmodern/.local/lib/pd/pof/scripts/ubuntu/../../apothecary: No such file or directory
./install_dependencies.sh: line 177: ./apothecary: No such file or directory
chown: cannot access 'build/poco': No such file or directory
chown: cannot access '../../libs/poco': No such file or directory
```

*PS. Purr-data simply crashes with a SIGINT when putting `pof` on the lib list.*

**Comment by Ant1r** [comment]
Hi,
please could you help me finding which lib is missing, and prevents deken's pof to work in your system ?

Run `pd -verbose -lib pof`, then switch Pd console Log level to 4 ("all"); Pd console then should tell you which library is missing, and hopefully I'll be able to update deken's pof.

Thanks!

**Comment by neopostmodern** [comment]
The pd-console said
```
'pd-gui' connecting to 'pd' on localhost 5400 ...
------------------ done with main ----------------------
Default font: DejaVu Sans Mono
tried ./pof.l_ia64 and failed
tried ./pof.pd_linux and failed
tried ./pof/pof.l_ia64 and failed
tried ./pof/pof.pd_linux and failed
tried ./pof.pd and failed
tried ./pof.pat and failed
tried ./pof/pof.pd and failed
tried /home/neopostmodern/.local/lib/pd/pof.l_ia64 and failed
tried /home/neopostmodern/.local/lib/pd/pof.pd_linux and failed
tried /home/neopostmodern/.local/lib/pd/pof/pof.l_ia64 and succeeded
/home/neopostmodern/.local/lib/pd/pof/pof.l_ia64: libpng12.so.0: cannot open shared object file: No such file or directory
tried /home/neopostmodern/.local/lib/pd/pof.pd and failed
tried /home/neopostmodern/.local/lib/pd/pof.pat and failed
tried /home/neopostmodern/.local/lib/pd/pof/pof.pd and failed
tried /home/neopostmodern/.local/lib/pd/extra/pof.l_ia64 and failed
tried /home/neopostmodern/.local/lib/pd/extra/pof.pd_linux and failed
tried /home/neopostmodern/.local/lib/pd/extra/pof/pof.l_ia64 and failed
tried /home/neopostmodern/.local/lib/pd/extra/pof/pof.pd_linux and failed
tried /home/neopostmodern/.local/lib/pd/extra/pof.pd and failed
tried /home/neopostmodern/.local/lib/pd/extra/pof.pat and failed
tried /home/neopostmodern/.local/lib/pd/extra/pof/pof.pd and failed
tried /home/neopostmodern/pd-externals/pof.l_ia64 and failed
tried /home/neopostmodern/pd-externals/pof.pd_linux and failed
tried /home/neopostmodern/pd-externals/pof/pof.l_ia64 and failed
tried /home/neopostmodern/pd-externals/pof/pof.pd_linux and failed
tried /home/neopostmodern/pd-externals/pof.pd and failed
tried /home/neopostmodern/pd-externals/pof.pat and failed
tried /home/neopostmodern/pd-externals/pof/pof.pd and failed
tried /usr/local/lib/pd-externals/pof.l_ia64 and failed
tried /usr/local/lib/pd-externals/pof.pd_linux and failed
tried /usr/local/lib/pd-externals/pof/pof.l_ia64 and failed
tried /usr/local/lib/pd-externals/pof/pof.pd_linux and failed
tried /usr/local/lib/pd-externals/pof.pd and failed
tried /usr/local/lib/pd-externals/pof.pat and failed
tried /usr/local/lib/pd-externals/pof/pof.pd and failed
tried /usr/lib/puredata/extra/pof.l_ia64 and failed
tried /usr/lib/puredata/extra/pof.pd_linux and failed
tried /usr/lib/puredata/extra/pof/pof.l_ia64 and failed
tried /usr/lib/puredata/extra/pof/pof.pd_linux and failed
tried /usr/lib/puredata/extra/pof.pd and failed
tried /usr/lib/puredata/extra/pof.pat and failed
tried /usr/lib/puredata/extra/pof/pof.pd and failed
tried /usr/lib/pd/extra/pof.l_ia64 and failed
tried /usr/lib/pd/extra/pof.pd_linux and failed
tried /usr/lib/pd/extra/pof/pof.l_ia64 and failed
tried /usr/lib/pd/extra/pof/pof.pd_linux and failed
tried /usr/lib/pd/extra/pof.pd and failed
tried /usr/lib/pd/extra/pof.pat and failed
tried /usr/lib/pd/extra/pof/pof.pd and failed
pof: can't load library
Loading plugin: /usr/lib/puredata/tcl/pd_deken.tcl
Loading plugin: /usr/lib/pd/extra/deken-plugin/deken-plugin.tcl
[deken]: installed version [0.2.1] == 0.2.1...skipping!
Loading plugin: /usr/lib/pd/extra/deken-xtra-apt-plugin/deken-xtra-apt-plugin.tcl
opened 0 MIDI input device(s) and 0 MIDI output device(s).
The Pd window filtered 58 lines
The Pd window filtered 59 lines
```
Thus I downloaded [the package from the archives](http://cz.archive.ubuntu.com/ubuntu/pool/main/libp/libpng/libpng12-0_1.2.54-1ubuntu1_amd64.deb) (as suggested [here](https://askubuntu.com/questions/915477/install-libpng12-on-17-04#comment1442910_915477)) and installed it.

Then I got another error complaining about `libwebpmux`/`libwebp` which I fixed with symlinks:
```
sudo ln -s /usr/lib/x86_64-linux-gnu/libwebpmux.so.2 /usr/lib/x86_64-linux-gnu/libwebpmux.so.1
sudo ln -s /usr/lib/x86_64-linux-gnu/libwebp.so.6 /usr/lib/x86_64-linux-gnu/libwebp.so.5
```

Now it seems to work with both vanilla and purr! No idea how to properly put this into a script though.

**Comment by Ant1r** [comment]
So I'm updating the upload.sh scripts, in order to include those libs also. This will take effect on next upload to deken (not sure when though).
Thanks !

**Comment by Ant1r** [comment]
This issue should be fixed with latest upload v0.1.7.
